testsuite/features/build_validation/core/srv_disable_default_schedule_tasks.feature

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Remove schedule tasks to manually trigger them when we consider the best moment
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Delete scheduled reposyncs
+    When I follow the left menu "Admin > Task Schedules"
+    And I follow "mgr-sync-refresh-default"
+    And I choose "disabled"
+    And I click on "Update Schedule"
+    And I click on "Delete Schedule"
+
+  Scenario: Delete scheduled sync of custom channels
+    When I follow the left menu "Admin > Task Schedules"
+    And I follow "channel-repodata-default"
```

Comment:
This task stop generation of metadata and not sync of custom channels

Reply:
The [documentation](https://documentation.suse.com/suma/4.3/en/suse-manager/administration/task-schedules.html) says the same:
![image](https://github.com/uyuni-project/uyuni/assets/12104291/7fa6d27a-ac74-4d3b-b019-0724e52e3a62)

Reply:
Yes, disabling the `mgr-create-bootstrap-repo` after reposync will solve a lot of our issues in BV. Could you share with me how to disable this behaviour ?

Reply:
Set in rhn.conf:
server.susemanager.auto_generate_bootstrap_repo = 0